.github/workflows/build-and-test.yml

```diff
@@ -183,7 +181,7 @@ jobs:
         run: ./gradlew --build-cache :integration:runTest --args="suites/FullIntegration.json /tmp/foo"
 
   notify-slack:
-    needs: [ build, jib, unit-tests, source-clear, sonar-scanner, integration-tests ]
+    needs: [ build, jib, source-clear, sonar-scanner, integration-tests ]
```

Comment:
I think it's fine to keep `unit-tests` here, that way if we move stuff around we don't need to manually figure out the dependency tree (otherwise for consistency should we also remove `build`?)

Reply:
Good point, I'll add it back